src/sliceDataStorage.h

```diff
@@ -208,6 +210,18 @@ class SliceMeshStorage : public SettingsMessenger // passes on settings from a M
      * \return whether a particular extruder is used by this mesh on a particular layer
      */
     bool getExtruderIsUsed(int extruder_nr, int layer_nr) const;
+
+    /*!
+     * \return the mesh's user specified z seam hint
```

Comment:
I like that you use the word 'hint' here.
Makes it a lot more clear what this is! :+1: